src/icons/iconTypes.ts

```diff
@@ -0,0 +1,36 @@
+import { ReactElement } from 'react';
+
+export interface TwoTonedIcon {
```

Comment:
As a note, we may be moving away from these types entirely (with the new assets, we won't have any that allow for two colors to be set.

This looks good, tho!